__docs__/wonder-blocks-typography/body-text.stories.tsx

```diff
@@ -161,29 +161,66 @@ export const ClassicConversionGuide = {
     render: () => (
         <View style={[styles.grid, styles.conversionGuide]}>
             <View style={styles.row}>
-                <LabelXSmall tag="p">LabelXSmall</LabelXSmall>
+                <LabelXSmall tag="p" style={styles.classic}>
+                    LabelXSmall
+                </LabelXSmall>
                 <BodyText size="xsmall">BodyText size=xsmall</BodyText>
             </View>
             <View style={styles.row}>
-                <LabelSmall tag="p">LabelSmall</LabelSmall>
+                <LabelSmall tag="p" style={styles.classic}>
+                    LabelSmall
+                </LabelSmall>
                 <BodyText size="small">BodyText size=small</BodyText>
             </View>
             <View style={styles.row}>
-                <LabelMedium tag="p">LabelMedium</LabelMedium>
+                <LabelMedium tag="p" style={styles.classic}>
+                    LabelMedium
+                </LabelMedium>
                 <BodyText>BodyText</BodyText>
             </View>
             <View style={styles.row}>
-                <LabelLarge tag="p">LabelLarge</LabelLarge>
+                <LabelLarge tag="p" style={styles.classic}>
+                    LabelLarge
+                </LabelLarge>
                 <BodyText weight="bold">BodyText weight=bold</BodyText>
             </View>
             <View style={styles.row}>
-                <Body tag="p">Body</Body>
+                <Body tag="p" style={styles.classic}>
+                    Body
+                </Body>
                 <BodyText>BodyText</BodyText>
             </View>
         </View>
     ),
 };
 
+/**
+ * An example of overriding `BodyText` component's styling.
+ */
+export const CustomStyling = {
```

Comment:
Do you think it's helpful to have Chromatic tests for this story for both TB and Classic themes? I was thinking one of them should be enough since overriding works the same way for both themes. Will leave it up to you!